quipucords/api/migrations/0023_migrate_systemfingerprint_sources_to_jsonfield.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.17 on 2023-03-16 15:18
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("api", "0022_migrate_metadata_to_jsonfield"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="systemfingerprint",
+            name="sources",
+            field=models.JSONField(null=True),
```

Comment:
~I'm curious about migrations like this one.
with the change to default shouldn't they change as well?~

EDIT I did run the branch locally and there was no need to update migrations. This is interesting. So this means that "default" sets db to accept null and the default object is handled on python side. Would be nice if there's an option to do that at db level (at times like these I miss sqlalchemy)

Reply:
Yeah, I updated the migrations here in commit [f543d61a](https://github.com/quipucords/quipucords/pull/2276/commits/f543d61a0fe8adfb524ac6678623a73de8f116a5)  as makemigrations wanted to see those changes. Otherwise, they will be reflected in any future run.